src/proto/tactic.proto

```diff
@@ -28,7 +28,7 @@ message Tactic
         PivotKickTactic pivot_kick                          = 11;
         ReceiverTactic receiver                             = 12;
         ShadowEnemyTactic shadow_enemy                      = 13;
-        StopTactic stop                                     = 14;
+        HaltTactic stop                                     = 14;
```

Comment:
This should be ideally changed to `halt`

Reply:
fixed